CHANGES_NEXT_RELEASE

```diff
@@ -5,3 +5,4 @@ Add: monit_log_processing.py script to contextBroker RPM (Issue #1083)
 Add: httpsPrepare.sh script to contextBroker-tests RPM (Issue #2)
 Fix:  Fixed a bug about generic error handling for API version 2 (Issue #1087)
 Fix:  Fixed a bug about error handling of invalid Service-Path for API version 2 (Issue #1092)
+Add:  Implemented a RAM-cache for patterned subscriptions (Issue #1048)
```

Comment:
Maybe "patterned NGSI10 subscriptions" would be more clear (to avoid missunderstanding with NGSI9 subscriptions, which don't use the cache).